substrate/bin/node/runtime/src/lib.rs

```diff
@@ -1587,6 +1587,20 @@ impl pallet_custom_account::Config<pallet_custom_account::Instance1> for Runtime
 	type Currency = Balances;
 }
 
+parameter_types! {
+	pub const SenateAccountPalletId: PalletId = PalletId(*b"lltreasu");
```

Comment:
This is the same ID as LLM treasury: https://github.com/liberland/liberland_substrate/blob/main/substrate/frame/llm/src/lib.rs#L636
It means, that the account backing this account will be `5EYCAe5hveooUENA5d7dwq3caqM4LLBzktNumMKmhNRXu4JE`, same as the treasury, so it will be able to control all funds that are already there on our prod networks.